hasher-matcher-actioner/terraform/main.tf

```diff
@@ -255,6 +228,40 @@ module "authentication" {
   webapp_and_api_shared_user_pool_client_id = var.webapp_and_api_shared_user_pool_client_id
 }
 
+
+/**
+ * # Primary S3 Bucket:
+ * Jack-of-all-trades S3 bucket. Used for storing raw data from threatexchange,
+ * checkpoints, and upload-type media submissions.
+ *
```

Comment:
nit: acknowledge the existence of `"aws_s3_bucket_notification" "bucket_notifications" {` and what it does here as it is now harder to trace and easier to miss, I would almost recommend pull that out of `hashing-data` because of this but am okay with you punting refactor changes like that for a future PR.

Reply:
I am kind of getting confused with what hashing-data intends to do. Perhaps in a future commit, I can move the SQS events to the hasher submodule, because that's what it is: an entry point to the hasher.

note: I have added an acknowledgement in the feedback commit.